src/pykeen/constants.py

```diff
@@ -2,13 +2,24 @@
 
 """Constants for PyKEEN."""
 
-import os
+import pystow
 
 __all__ = [
     'PYKEEN_HOME',
+    'PYKEEN_DATASETS',
+    'PYKEEN_BENCHMARKS',
+    'PYKEEN_EXPERIMENTS',
+    'PYKEEN_CHECKPOINTS',
 ]
 
-PYKEEN_HOME = os.environ.get('PYKEEN_HOME') or os.path.join(os.path.expanduser('~'), '.pykeen')
+#: Can be modified by setting environment variable ``PYKEEN_HOME``
+#: For more information, see https://github.com/cthoyt/pystow
+PYKEEN_HOME = pystow.get('pykeen')
+PYKEEN_DATASETS = pystow.get('pykeen', 'datasets')
+PYKEEN_BENCHMARKS = pystow.get('pykeen', 'benchmarks')
+PYKEEN_EXPERIMENTS = pystow.get('pykeen', 'experiments')
+PYKEEN_CHECKPOINTS = pystow.get('pykeen', 'checkpoints')
```

Comment:
These are just the defaults for these locations and can be changed if the user provides something else, right?

Reply:
They’ll all adjust to be inside PYKEEN_HOME automatically if that’s set in the environment